src/mobu/jupyterclient.py

```diff
@@ -158,98 +165,130 @@ async def lab_login(self) -> None:
             if r.status != 200:
                 await self._raise_error("Error logging into lab", r)
 
-    async def is_lab_running(self) -> bool:
-        hub_url = self.jupyter_url + "hub"
-        spawn_url = self.jupyter_url + "hub/spawn"
-        spawn_pending_url = (
-            self.jupyter_url + f"hub/spawn-pending/{self.user.username}"
-        )
-        lab_url = self.jupyter_url + f"user/{self.user.username}/lab"
-
-        async with self.session.get(hub_url) as r:
+    async def lab_is_stopped(self) -> bool:
```

Comment:
This is really bletcherous.  I get why it's necessary.  I will go ask on the gitter whether this is already an extant API, and if the answer is "er, no" then maybe creating that and submitting the PR should be a priority for us, because surely we're not the only people who need it.

Reply:
Thank you! Yeah, it seems obvious there should be an API for this.